cfpb/regulations-site
# Remove Bower dependency

This moves us to completely using `npm` to manage packages on the front-end.

To test:

1. Clone my branch locally and run the `frontendbuild.sh` script. This will run `npm install` and the `grunt build` command. The site should build with no errors.
2. Start the server and ensure that there are no console.log errors reporting unfound scripts.

## Review

@willbarton
